shared/constants/field/myinfo/index.ts

```diff
@@ -395,7 +395,7 @@ export const types: MyInfoFieldBlock[] = [
 
 export const MYINFO_ATTRIBUTE_MAP = keyBy(types, 'name')
 
-// to remove after 1 July. For testing purposes, deliberately setting target date as 1 day before today (10 Jun) to simulate change
+// TO DO: 1 July. For testing purposes, deliberately setting target date as 1 day before today (10 Jun) to simulate change
```

Comment:
Sorry! Didn't want to look like I'm nitpicking but there's a difference if we have a space between `to` and `do`

<img width="447" alt="Screenshot 2024-06-12 at 11 20 24 PM" src="https://github.com/opengovsg/FormSG/assets/12391617/0e652a03-2367-49f3-8321-20282c69c175">

Reply:
updated, thanks for letting me know @KenLSM ! (: